tools/flaky-test-reporter/ghutil/ghutil.go

```diff
@@ -0,0 +1,327 @@
+	return updateIssueState(org, repo, IssueOpenState, number)
+}
+
+// ListComments gets all comments from issue
+func ListComments(org, repo string, number int) ([]*github.IssueComment, error) {
+	var res []*github.IssueComment
+	options := &github.ListOptions{}
+	genericList, err := depaginate(
+		fmt.Sprintf("commenting issue '%s %s %d'", org, repo, number),
+		maxRetryCount,
+		options,
+		func() ([]interface{}, *github.Response, error) {
+			page, resp, err := client.Issues.ListComments(ctx, org, repo, number, nil)
+			var interfaceList []interface{}
+			if nil == err {
+				for _, issue := range page {
+					interfaceList = append(interfaceList, issue)
+				}
+			}
+			return interfaceList, resp, err
+		},
+	)
+	for _, issue := range genericList {
+		res = append(res, issue.(*github.IssueComment))
+	}
+	return res, err
+}
+
+// GetComment gets comment by comment ID
+func GetComment(org, repo string, commentID int64) (*github.IssueComment, error) {
+	var res *github.IssueComment
+	_, err := retry(
+		fmt.Sprintf("getting comment '%s %s %d'", org, repo, commentID),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			res, resp, err = client.Issues.GetComment(ctx, org, repo, commentID)
+			return resp, err
+		},
+	)
+	return res, err
+}
+
+// CreateComment adds comment to issue
+func CreateComment(org, repo string, number int, commentStr string) error {
+	comment := &github.IssueComment{
+		Body: &commentStr,
+	}
+	_, err := retry(
+		fmt.Sprintf("commenting issue '%s %s %d'", org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.CreateComment(ctx, org, repo, number, comment)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// EditComment edits comment with given comment ID
+func EditComment(org, repo string, commentID int64, commentStr string) error {
+	comment := &github.IssueComment{
+		Body: &commentStr,
+	}
+	_, err := retry(
+		fmt.Sprintf("editing comment '%s %s %d'", org, repo, commentID),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.EditComment(ctx, org, repo, commentID, comment)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// AddLabelsToIssue adds label on issue
+func AddLabelsToIssue(org, repo string, number int, labelStrs []string) error {
+	_, err := retry(
+		fmt.Sprintf("add labels '%v' to '%s %s %d'", labelStrs, org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.AddLabelsToIssue(ctx, org, repo, number, labelStrs)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// RemoveLabelForIssue in case test failed, remove label auto:lessflaky if exists
+func RemoveLabelForIssue(org, repo string, number int, labelStr string) error {
```

Comment:
s/number/issueNumber/

I also think that just "label" is cleaner and offers better readability.

Reply:
done. label as well